<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1501</width>
    <height>863</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QTabWidget" name="tabWidget_viewer">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_15">
              <attribute name="title">
               <string>synoptic viewer</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QGridLayout" name="gridLayout_14">
                 <item row="0" column="7">
                  <widget class="QPushButton" name="pushButton_mv_viewer_left">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>mv left</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="9">
                  <widget class="QPushButton" name="pushButton_mv_viewer_right">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>mv right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="8">
                  <widget class="QSpinBox" name="spinBox_step_size">
                   <property name="minimum">
                    <number>5</number>
                   </property>
                   <property name="maximum">
                    <number>500</number>
                   </property>
                   <property name="singleStep">
                    <number>5</number>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="10">
                  <widget class="QLabel" name="label_89">
                   <property name="text">
                    <string>squeeze view</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="11">
                  <widget class="QSlider" name="horizontalSlider_viewer_squeezer">
                   <property name="minimum">
                    <number>5</number>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="pageStep">
                    <number>2</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="6">
                  <widget class="QLabel" name="label_75">
                   <property name="text">
                    <string>mv viewer</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QPushButton" name="pushButton_render">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Render Viewer</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboBox_viewer_filename">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QComboBox" name="comboBox_viewer_obj_name">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_80">
                   <property name="text">
                    <string>Viewer objects</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_79">
                   <property name="text">
                    <string>Viewer files</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSplitter" name="splitter_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QFrame" name="frame_10">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <item>
                    <widget class="beamlineSynopticViewer" name="widget_synoptic" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>280</height>
                      </size>
                     </property>
                     <property name="mouseTracking">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QFrame" name="frame_9">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_17">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_19">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="text">
                          <string>camera source</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBox_cam_source1"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="Taurus2DImageItem" name="widget_camera_viewer1" native="true"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_13">
                       <item>
                        <widget class="QLabel" name="label_29">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="text">
                          <string>camera source</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBox_cam_source2"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="Taurus2DImageItem" name="widget_camera_viewer2" native="true"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_14">
                       <item>
                        <widget class="QLabel" name="label_74">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="text">
                          <string>camera source</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBox_cam_source3"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="Taurus2DImageItem" name="widget_camera_viewer3" native="true"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Config</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QFrame" name="frame_11">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <widget class="SmartParameters" name="widget_pars" native="true">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_18">
                    <item>
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_load_config">
                      <property name="text">
                       <string>Load</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_apply_change">
                      <property name="text">
                       <string>Apply</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_24"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="maximumSize">
           <size>
            <width>9000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Console</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="TerminalWidget" name="widget_terminal" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TerminalWidget</class>
   <extends>QWidget</extends>
   <header>smart_shape.gui.widgets.terminal_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>beamlineSynopticViewer</class>
   <extends>QWidget</extends>
   <header>smart_shape.gui.widgets.shape_viewer_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Taurus2DImageItem</class>
   <extends>QWidget</extends>
   <header>smart_shape.gui.widgets.camera_viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmartParameters</class>
   <extends>QWidget</extends>
   <header>smart_shape.gui.widgets.parameter_tree.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
